doc/Caliptra.md

```diff
@@ -916,7 +916,7 @@ As shown in *Table 15: POST/CAST usage*, since the cryptographic algorithms requ
 
 ## FW Signing/Verification algorithms
 
-Caliptra firmware is composed of multiple layers: an FMC and an application firmware image. Each layer is signed individually by ECDSA P384 keys.
+Caliptra firmware is composed of multiple layers: an FMC and an application firmware image. Each layer is signed individually by ECDSA P384 keys, and optionally [LMS Signatures](#post-quantum-cryptography-pqc-requirements) as well.
 
 Each layer is signed by a vendor-controlled key. In addition, each layer may also be signed by an owner-controlled key. The image header contains both the owner public key and the signature using that key.
```

Comment:
noticed that this sentence is not correct, there's only one signature set for all firmware. but outside the scope of your PR